packages/core/src/layout/ItemCard/ItemCard.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { FC } from 'react';
+import { Button, Card, Chip, Typography, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  header: {
+    color: theme.palette.common.white,
+    padding: theme.spacing(2, 2, 6),
+    backgroundImage:
+      'linear-gradient(-137deg, rgb(25, 230, 140) 0%, rgb(29, 127, 110) 100%)',
+  },
+  content: {
+    padding: theme.spacing(2),
+  },
+  description: {
+    height: 175,
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+  },
+  footer: {
+    display: 'flex',
+    flexDirection: 'row-reverse',
+  },
+}));
+
+type ItemCardProps = {
+  description: string;
+  tags?: string[];
+  title: string;
+  type?: string;
+  label: string;
+  onClick?: () => void;
+};
+export const ItemCard: FC<ItemCardProps> = ({
```

Comment:
What's an `ItemCard`, and how do you choose between using it and an `InfoCard`? 😅

Reply:
We were really just looking through the different Cards that already exist, InfoCard, TemplateCard, ExploreCard etc and decided to create a reusable one instead of having one more Card that could be iterated on to cover more use cases. 
Would you say InfoCard is a better component to use for this use case as well?

Reply:
Agree the naming can be changed though..

Reply:
> What's an `ItemCard`, and how do you choose between using it and an `InfoCard`? sweat_smile
ItemCard is basically TemplateCard pulled out into core. It's more or less the same design of card for ExploreCard and TemplateCard so we didn't want to make a third used in only one place. Hoped this would create some discussion around a common one.


![2020-06-29-112506_612x445_scrot](https://user-images.githubusercontent.com/701797/86000247-94ae1800-ba0d-11ea-98c5-50d5befb1996.png)

Reply:
I'd say stick to using `InfoCard` if possible. If not I think you need to be able to document what the use-case for the `ItemCard` is and when to use it. Calling in @katz95 too though, as I don't know the current thinking around different card-type components.

Reply:
Thanks @Rugvip we will move that component into the techdocs plugin for now, so we can reuse it and then we can sync on the different card-type components you are planning for!

Reply:
So figured that this is the card on the explore page, which is a pretty well defined separate use-case, so makes sense to have. 👍   to adding the card, but with some more docs explaining when to use it, as well as a storybook that shows it in the grid-view way of using it.

Reply:
Alright, cool! We will separate it out again then, in a next upcoming PR in a few min. Thanks!